iml-agent/src/action_plugins/ltuer.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::agent_error::ImlAgentError;
+use std::path::{Path, PathBuf};
+use tokio::{
+    fs,
+    io::{AsyncWrite, AsyncWriteExt},
+};
+
+static CONFIGURATION_DIR: &str = "/etc/iml";
```

Comment:
We should probably add this as an environment variable to https://github.com/whamcloud/integrated-manager-for-lustre/blob/402a75118afbd232c861273a47b9b7bf1b8396e1/iml-agent/systemd-units/rust-iml-agent.service#L7-L17


That way it's configurable outside the code, and can be used by all agent modules.

Reply:
Done